playbooks/de-rm-containers-cleanup.yaml

```diff
@@ -45,3 +45,41 @@
       ignore_errors: yes
       become: true
       shell: docker rm -v config_jexevents
+
+- name: Remove anon-files and anon-files-nginx (CORE-8534)
+  hosts: systems
```

Comment:
(this is broader than needed, but makes sure it cleans it up, since I'm also removing the anon-files host group)